source/Command.h

```diff
@@ -125,6 +125,7 @@ class Command {
 private:
 	// The key commands and weapons to fire are stored in a single bitmask, with
 	// 32 bits for key commands and 32 bits for individual weapons.
+	// Ship::Load gives a soft warning for ships with more than 32 weapons.
```

Comment:
I added this comment by purpose.
Because you could decide to change from 64bit int to std::bitset with a greater length and then it would be wise to adjust that comment.
(I prefer to document dependencies if there are only a few)